pkg/service/events_test.go

```diff
@@ -52,7 +53,7 @@ func Test_PublishEvent(t *testing.T) {
 	publishKubernetesEvent(kubeClient, event)
 	expectedEvents := 1
 
-	events, err := kubeClient.CoreV1().Events(EventNamespace).List(metav1.ListOptions{})
+	events, err := kubeClient.CoreV1().Events(EventNamespace).List(context.Background(), metav1.ListOptions{})
```

Comment:
Just curious, how does adding context help here?

Reply:
It is required as a part of an update to apimachinery module.